src/Samples/FileDialogs.CmdMsg/Program.fs

```diff
@@ -113,17 +116,19 @@ open Core
 open Platform
 
 
+let vm = ViewModel.designInstance (initModel ()) (bindings ())
```

Comment:
Instead of factoring out an `initModel`, how do you feel about just using `(init () |> fst)` in the call to `designInstance`? Keeps the rest of the code a bit cleaner, I think.

Reply:
Oh, sure.  I didn't even consider that.